packages/components/src/components/hive-navigation/index.tsx

```diff
@@ -0,0 +1,562 @@
+                      <Logo className="size-8 text-green-1000 dark:text-neutral-300" />
+                    </div>
+                    <div>
+                      <p className="text-base font-medium leading-normal text-green-1000 dark:text-neutral-200">
+                        {product.name}
+                      </p>
+                    </div>
+                  </NavigationMenuLink>
+                </li>
+              );
+            })}
+          </ul>
+          <Anchor
+            href={EXPLORE_HREF}
+            className="-my-2 ml-2 flex items-center gap-2 rounded-lg p-2 font-medium text-green-800 transition-colors hover:bg-beige-100 hover:text-green-1000 dark:text-neutral-400 dark:hover:bg-neutral-800/50 dark:hover:text-neutral-200"
+          >
+            <span>Explore all libraries</span> <ArrowIcon />
+          </Anchor>
+        </div>
+      </MenuContentColumns>
+    );
+  },
+);
+ProductsMenu.displayName = 'ProductsMenu';
+
+interface MenuContentColumnsProps extends React.HTMLAttributes<HTMLDivElement> {}
+const MenuContentColumns = forwardRef(
+  (props: MenuContentColumnsProps, ref: React.ForwardedRef<HTMLDivElement>) => {
+    return (
+      <div className="flex gap-x-6 [&>*]:flex [&>*]:flex-col [&>*]:gap-4" ref={ref} {...props}>
+        {React.Children.toArray(props.children)
+          .filter(Boolean)
+          .map((child, index, array) => {
+            return (
+              <Fragment key={index}>
+                {child}
+                {index < array.length - 1 && (
+                  <div className="w-px bg-beige-200 dark:bg-neutral-800" />
+                )}
+              </Fragment>
+            );
+          })}
+      </div>
+    );
+  },
+);
+MenuContentColumns.displayName = 'MenuContentColumns';
+
+interface DeveloperMenuProps extends React.HTMLAttributes<HTMLDivElement> {
+  isHive: boolean;
+}
+/**
+ * @internal
+ */
+export const DeveloperMenu = React.forwardRef<HTMLDivElement, DeveloperMenuProps>(
+  ({ isHive, ...rest }, ref) => {
+    return (
+      <MenuContentColumns {...rest} ref={ref}>
+        <div>
+          <ColumnLabel>Developer</ColumnLabel>
+          <ul>
+            {(
+              [
+                [
+                  'Documentation',
+                  PaperIcon,
+                  isHive ? '/docs' : 'https://the-guild.dev/graphql/hive/docs',
+                ],
+                ['Status', TargetIcon, 'https://status.graphql-hive.com/'],
+                [
+                  'Product Updates',
+                  RightCornerIcon,
+                  isHive
+                    ? '/product-updates'
+                    : 'https://the-guild.dev/graphql/hive/product-updates',
+                ],
+                ['Blog', PencilIcon, 'https://the-guild.dev/blog'],
+                ['GitHub', GitHubIcon, 'https://github.com/kamilkisiela/graphql-hive'],
+              ] as const
```

Comment:
use object, not tuple

Reply:
This is immediately mapped into JSX, so any property keys just bloat the bundle — this is why I just used tuples here.

Do you guys have a convention against tuples? Also cc @saihaj because I think he gave me the same comment elsewhere.

I don't mind changing it, I'm just curious.